pkgs/applications/networking/instant-messengers/slack/default.nix

```diff
@@ -73,7 +73,8 @@ in stdenv.mkDerivation {
 
     # Fix the symlink
     rm $out/bin/slack
-    ln -s $out/lib/slack/slack $out/bin/slack
+    makeWrapper $out/lib/slack/slack $out/bin/slack \
+      --set LD_LIBRARY_PATH ${rpath}
```

Comment:
And I guess `--set` as opposed to addition means that OpenGL drivers are not found. It is an Electron application, so it would use GPU to offload part of the rendering, right?